easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.1-foss-2016a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,24 @@
+name = 'SuiteSparse'
+version = '4.5.1'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
+
+preconfigopts = 'LAPACK="$LIBLAPACK" ' 
+prebuildopts = 'LAPACK="$LIBLAPACK" ' 
+preinstallopts = 'LAPACK="$LIBLAPACK" ' 
```

Comment:
After a second check, it appears the library is linked to both `libopenblas.so` and `libblas.so`.
```
$ ldd $EBROOTSUITESPARSE/lib/libumfpack.so
...
        liblapack.so.3 => /usr/lib/liblapack.so.3 (0x00007f2595e63000)
        libopenblas.so.0 => /home/users/xbesseron/.local/easybuild/software/numlib/OpenBLAS/0.2.15-GCC-4.9.3-2.25-LAPACK-3.6.0/lib/libopenblas.so.0 (0x00007f25951c5000)
        libblas.so.3 => /usr/lib/libblas.so.3 (0x00007f2594416000)
...
```

This happens only for `foss`. It appears to be OK for `ictce`.